packaging/usr/share/bash-completion/completions/pikaur

```diff
@@ -50,6 +50,51 @@ _pacman_pkg() {
   )"
 }
 
+_pikaur_get_aur_packages() {
+  local cacheFile=$1
+  local aurPackagesURL='https://aur.archlinux.org/packages.gz'
+  local tmpFile
+  tmpFile="$(mktemp)"
+  curl -f -s "${aurPackagesURL}" -o "${tmpFile}"
+  local returnStatus=$?
+  if [ "${returnStatus}" -eq 0 ]; then
+    mv "${tmpFile}" "${cacheFile}" 2>/dev/null
+  else
+    rm "${tmpFile}"
+  fi
+}
+
+_pikaur_cached_search() {
+  local repoPackageNames
+  repoPackageNames="$(\pacman -Ssq)" # pacman is faster then pikaur -Ssqo
+  local cacheDir
+  if [ -n "${XDG_CACHE_HOME+x}" ]; then
+    cacheDir="${XDG_CACHE_HOME}/pikaur"
+  else
+    cacheDir="${HOME}/.cache/pikaur"
```

Comment:
please dont use pikaur's own subdirectory

Reply:
I don't understand why I shouldn't use `~/.cache/pikaur` as a directory for cache.

Reply:
because it's not produced by pikaur itself,
use separate dir or file for pikaur bash completion cache

Reply:
List of all options
1. `~/.cache/pikaur/`
1. `~/.cache/pikaur/bash-completion/`
1. `~/.cache/pikaur_bash-completion/`
1. `~/.cache/pikaur_bash_completion/`

As I understand options 1 and 2 are unacceptable.
Then I should probably use `~/.cache/pikaur_bash-completion`. Is this ok?

Personally I don't want to have files in my `~/.cache` dir.
https://github.com/actionless/pikaur/blob/0f3786d33884bc79082fd9b1d83072f87cf39560/packaging/usr/share/fish/completions/pikaur.fish#L23

Reply:
Changed to `local cacheDir="${XDG_CACHE_HOME:-$HOME/.cache}/pikaur_bash-completion"`